The lack of a PREFIX or X11 path variable, the problems with the install
target, and the duplicate lines in the Darwin section have been reported
to the developer by email.
--- makefile.orig	2021-04-07 13:48:56.000000000 -0500
+++ makefile	2021-04-25 10:47:53.000000000 -0500
@@ -16,7 +16,7 @@
 CC = cc
 LIBS =  -lX11 -lm
 LDR = ${FC} 
-LIBSG = -L/usr/X11R6/lib -lGLU -lGL -lXmu -lX11 -lm
+LIBSG = -L$(PREFIX)/lib -lGLU -lGL -lXmu -lX11 -lm
 ARCH := $(shell getconf LONG_BIT)
 AFLAG= -m$(ARCH)
 EXTEN=
@@ -131,15 +131,11 @@
 # Mac OS X g77
 #
 CC=cc
-CFLAGS= -std=gnu89 -Wno-return-type -DDARWIN -I/usr/X11R6/include -DDOBACK -DHASTIMER -DCOLOR_OFFSET=0.0 -Wno-logical-op-parentheses
-FFLAGS=-O3 -funroll-loops
-LIBS = -L/usr/X11R6/lib -lX11 -lm
-LIBSG = -L/usr/X11R6/lib -lGLU "-Wl,-dylib_file,/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib:/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib" -lGL -lXmu -lX11 -lm
 EXT= -std=gnu89 -Wno-return-type -DDARWIN ${EXTRAZ}
 FFLAGS=-O3 -funroll-loops
-LIBS = -L/usr/X11R6/lib -lX11 -lm
-LIBSG = -L/usr/X11R6/lib -lGLU "-Wl,-dylib_file,/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib:/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib" -lGL -lXmu -lX11 -lm
-CFLAGS= -g -std=gnu89 -Wno-return-type -DDARWIN -I/usr/X11R6/include -DDOBACK -DHASTIMER -DCOLOR_OFFSET=0.0 -Wno-logical-op-parentheses -Wno-tautological-pointer-compare -Wno-tautological-constant-out-of-range-compare
+LIBS = -L$(PREFIX)/lib -lX11 -lm
+LIBSG = -L$(PREFIX)/lib -lGLU "-Wl,-dylib_file,/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib:/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib" -lGL -lXmu -lX11 -lm
+CFLAGS= -g -std=gnu89 -Wno-return-type -DDARWIN -I$(PREFIX)/include -DDOBACK -DHASTIMER -DCOLOR_OFFSET=0.0 -Wno-logical-op-parentheses -Wno-tautological-pointer-compare -Wno-tautological-constant-out-of-range-compare
 LDR = ${FC}
 endif
 
@@ -270,7 +266,7 @@
 # and the full opengl version of molden 'gmolden', (does not need glut)
 # (make gmolden)
 #
-LIBSOGL = -lglut -lGLU -lGL -lXmu -lX11 -lm
+LIBSOGL = -L$(PREFIX)/lib -lglut -lGLU -lGL -lXmu -lX11 -lm
 #
 # on linux :
 #
@@ -363,7 +359,7 @@
 endif
 
 exten2:
-	utils/register_extension.sh /usr/local/bin
+	utils/register_extension.sh $(PREFIX)/bin
 
 install:	$(EXTENZ)
-	sudo install -t /usr/local/bin -m 755 bin/molden bin/gmolden bin/ambfor bin/ambmd bin/surf
+	install -m 755 bin/molden bin/gmolden bin/ambfor bin/ambmd bin/surf $(DESTDIR)$(PREFIX)/bin
